spec/integration/api/sys/quota_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "spec_helper"
+
+RSpec.shared_examples "quota specs" do |type|
```

Comment:
Should we include a set of tests for creating/updating and reading the config?

Reply:
Done